app/handlers/testConfig.py

```diff
@@ -70,6 +70,29 @@ def post(self):
             self.error_message = reason
             raise tornado.web.HTTPError(status_code=status_code)
 
+    def get(self):
+        status = StatusHelper.get_status(self.identifier)
+        if test_flag == "feature":
```

Comment:
test_flag is undefined, change this to self.test_flag and use it from prepare or create it somewhere